### PR TITLE
Add Abstract-CN.md

### PR DESCRIPTION
**Pull request by xinzhuxiansheng**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by xinzhuxiansheng** [comment]
hi,晟哥。"coverage/coveralls" told me that I failed, the "Details" are "UNCOVERED EXISTING LINES :  	apm-sniffer/apm-sdk-plugin/spring-plugins/core-patch/src/main/java/org/apache/skywalking/apm/plugin/spring/patch/AopExpressionMatchInterceptor.java", but I had just amended doc, I checked "Files changed" which is (README_ZH.md,Abstract-CN.md), I don't know why this happened.

**Comment by wu-sheng** [comment]
Don't worry about coverage.

**Comment by xinzhuxiansheng** [comment]
I'm so sorry that I cliked on the "Update branch" by accident, and I do not know what does that button means.

File: docs/cn/Incubating/Abstract-CN.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# 孵化特性
+孵化的功能特性是由SkyWalking社区人员开发和贡献代码.
+但目前SkyWalking仍然是雏形或预览版本, 社区希望有更多的开发者参与，给SkyWalking主分支贡献代码。
```
Comment (wu-sheng) [inline docs/cn/Incubating/Abstract-CN.md]: 这句话不通顺。应该是目前代码还是原型，或者预览版本，并已经提交到主干代码，并进行发布。同时，功能开发者希望社区更多的开发者参与，对功能进行完善。

File: docs/cn/Incubating/Abstract-CN.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+PMC和主开发团队会评估提交的特性，确保它是否具有移植性，是否兼容当前版本.
```
Comment (wu-sheng) [inline docs/cn/Incubating/Abstract-CN.md]: 应该是PMC和Committor团队

File: docs/cn/Incubating/Abstract-CN.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+列举一些典型的孵化功能特性
```
Comment (wu-sheng) [inline docs/cn/Incubating/Abstract-CN.md]: 下面是已知的，可能存在的孵化功能：

**Comment by xinzhuxiansheng** [comment]
hi,晟哥，i‘ve fixed some problems and i‘ve committed new doc [eceiver-zipkin/README-CN.md]

File: apm-collector/apm-collector-thirdparty-receiver/receiver-zipkin/docs/README-CN.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Zipkin接受器在SkyWalking中仅仅只是一个可选特性,即使到目前为止它依然是[孵化特性]
+(../../../../docs/cn/Incubating/Abstract-CN.md).
+
+## 局限性
+作为孵化功能,它仍是一个原型.所以它包含以下几点局限：
+
+1. 不要在同一分布式系统中使用SkyWalking原生探针和Zipkin的类库。考虑到Zipkin和SkyWalking的HEADER不是共享/可互操作的，它们的两个不会相互传播context.这会导致已跟踪的链接中断
+1. 不支持集群模式.
+1. 基于链路跟踪的分析将在特定时间内完成的.默认是不能超过2分钟.在分析阶段SkyWalking使用了更复杂的header和context来兼容处理Zipkin格式的数据.
```
Comment (wu-sheng) [inline apm-collector/apm-collector-thirdparty-receiver/receiver-zipkin/docs/README-CN.md]: 在进行链路分析时，基于链路会在制定时间内结束。默认链路执行时间不超过2分钟。SkyWalking使用了更复杂的header和context结构，来避免zipkin分析时的这个问题。

**Pull request merged**